backend/driver.pm

```diff
@@ -156,14 +154,13 @@ sub _send_json {
     my $token = myjsonrpc::send_json($self->{backend_process}->channel_in, $cmd);
     my $rsp   = myjsonrpc::read_json($self->{backend_process}->channel_out, $token);
 
-    unless (defined $rsp) {
-        # this might have been closed by signal handler
-        no autodie 'close';
-        close($self->{backend_process}->channel_out);
-        $self->{backend_process}->channel_out(undef);
-        $self->{backend_process}->stop;
-        return;
-    }
+    return undef if defined $rsp;
+    # this might have been closed by signal handler
+    no autodie 'close';
+    close($self->{backend_process}->channel_out);
+    $self->{backend_process}->channel_out(undef);
+    $self->{backend_process}->stop;
+    return;
     return $rsp->{rsp};
```

Comment:
This line is now unreachable

Reply:
that should explain the test failures :) Updated, thx.